### DISCUSSION TITLE
Building project errors

### DISCUSSION THREAD

**Discussion by lordrius** [FabricMC/fabric-api#3304] [Mod Dev Support] [answered]
### Description

Hi, I'm updating my mod to 1.20.2 snapshots and everything works perfectly, no errors, no crashes. But when I try to build the project, it throws a ton of errors that never happened before. I don't understand the meaning of the errors, it says cannot find symbol, package does not exist... I'm using Eclipse. Please help!

Thank you very much!

https://pastebin.com/7HDT5aEU

**Comment by lordrius** [comment]
Hi, I have tried with a fresh project. I'm using latest example-mod files. Using latest 1.20.2-pre2+build2 mappings and Fabric API 0.88.3.
It thows errors saying that mixin has no targets and cannot find packages. Something related to this issue https://github.com/FabricMC/fabric-example-mod/issues/104 but I already try to start with a fresh project, deleting .gradle folder.

Anyone?

PD: 1.20.1 builds perfectly

**Comment by modmuss50** [comment, marked as answer]
Are you using split sources (enabled in your build.gradle file). If so all of your client code must go into the client sourceset (src/client/java) and no the common sourceset. Another option would be to revert back to using a merged setup.

**Reply by lordrius** [reply]
Thank you very much modmuss, that's it, now it builds fine! :)
